open-company/open-company-web
# Digest routes

Card: https://trello.com/c/YX5srgup

With latest storage mainline:
- generate at least a couple of months of posts for an org
- navigate to a board that has some posts and scroll to the bottom
- take the last post and copy the `published-at` timestamp (look in dev tools Network tab and look for the headline if you don't know how to find it)
- visit: `/{org-slug}/{board-slug}/post/{post-uuid}?at={published-at}`
- [x] do you see the post in the modal? Good
- [x] do you see All Posts highlighted below the modal? Good
- now dismiss it
- [x] are you back to All Posts? Good
- [x] do you see the post with some more posts above it and some other below it (there may not be posts below if you choose the first created post)? Good
- [ ] deploy to beta
